tests/lcp/test_server.py

```diff
@@ -89,7 +89,7 @@ def test_add_content(self, _, input_directory):
         ('none_rights', None, None, None, None),
         (
                 'license_start',
-                datetime.datetime(2020, 01, 01, 00, 00, 00),
+                datetime.datetime(2020, 0o1, 0o1, 00, 00, 00),
```

Comment:
This works, but it's defining 1 as an octal number, which doesn't make sense and could cause problems in the future. I'm guessing this happened automatically as an attempt to make sense of `01`, which is not valid in Python 3. I think just change these to `1`.

Reply:
Yeah, this was an odd update from the converter.